sig/generated/rbs/inline/ast/annotations.rbs

```diff
@@ -44,6 +44,10 @@ module RBS
           # @rbs override
           def initialize: ...
 
+          @tree: untyped
+
+          @source: untyped
```

Comment:
attr_readers for these variables are defined in the base class. Therefore these definitions are bothersome...